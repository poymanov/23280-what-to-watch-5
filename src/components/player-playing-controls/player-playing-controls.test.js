import React from "react";
import renderer from "react-test-renderer";
import PlayerPlayingControls from "./player-playing-controls";

const noop = () => {};

describe(`Should PlayerPlayingControls render correctly`, () => {
  it(`Playing`, () => {
    const tree = renderer
      .create(<PlayerPlayingControls isPlaying={true} onClick={noop}/>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Paused`, () => {
    const tree = renderer
      .create(<PlayerPlayingControls isPlaying={false} onClick={noop}/>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});


