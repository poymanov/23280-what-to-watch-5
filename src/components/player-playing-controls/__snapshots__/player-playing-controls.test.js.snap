// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should PlayerPlayingControls render correctly Paused 1`] = `
<button
  className="player__play"
  onClick={[Function]}
  type="button"
>
  <svg
    height="19"
    viewBox="0 0 19 19"
    width="19"
  >
    <use
      href="#play-s"
    />
  </svg>
  <span>
    Play
  </span>
</button>
`;

exports[`Should PlayerPlayingControls render correctly Playing 1`] = `
<button
  className="player__play"
  onClick={[Function]}
  type="button"
>
  <svg
    height="21"
    viewBox="0 0 14 21"
    width="14"
  >
    <use
      href="#pause"
    />
  </svg>
  <span>
    Pause
  </span>
</button>
`;
